### ISSUE TITLE
Series / Movie drop down not working

### ISSUE THREAD

**Issue by londonreturns** [Stremio/stremio-web#560]
<img width="341" alt="Screenshot 2024-01-12 184149" src="https://github.com/Stremio/stremio-web/assets/76239421/39cd4d5a-5a95-476f-9bca-8178959e0c82">

Dropdown not working

**Comment by Ambushfall** [comment]
In my case, it's not working on win11 desktop client when clicking on Home > See All > Using Dropdown selection that comes from an addon.

Otherwise, going through Library > Dropdown works fine.

(The error I'm getting in my case is "Addon not installed").

Opening the web version works absolutely fine, without any single issue whatsoever.

Not sure if this is the right place to add this information or not.

**Comment by kKaskak** [comment]
Dropdown seems to be working totally fine on the web App. Maybe it happens when you are using the desktop app? Can you specify?

**Comment by londonreturns** [comment]
> Dropdown seems to be working totally fine on the web App. Maybe it happens when you are using the desktop app? Can you specify?

Yes it is the desktop app. It seems to be working fine now but then again it has happened already twice till now. I cannot recreate the bug.

**Comment by kKaskak** [comment]
For the desktop app you can use the https://github.com/Stremio/stremio-bugs to report issues. I will transfer this over there. 
Also please provide a screen recording when it happens again.
